website/docs/docs/build/incremental-models.md

```diff
@@ -310,5 +310,61 @@ select ...
 
 </File>
 
+<VersionBlock firstVersiont="1.4">
+
+### About incremental_predicates
+
+`incremental_predicates` is an advanced use of incremental models, where data volume is large enough to justify additional investments in performance. This config accepts any valid SQL expression, however dbt does not check the syntax. 
+
+For example, this is a pattern we might expect to see on Snowflake:
+
+```yml
+models:
+  - name: my_incremental_model
+    config:
+      materialized: incremental
+      unique_key: id
+      cluster_by: ['session_start']  # this will affect how the data is stored on disk, and indexed to limit scans
+      incremental_strategy: merge
+      # limit scan of the existing table to the last 7 days of data
+      incremental_predicates: "DBT_INTERNAL_DEST.session_start > datediff(day, -7, current_date)"
```

Comment:
will the user understand that they need to use DBT_INTERNAL_DEST`  or `DBT_INTERNAL_SOURCE`? or is this brand new specifications? i might have missed this but what do they mean?

Reply:
This is worth a call out! The decision was made to allow for users to write _any_ sql they want, and not over engineer to make sure we guess correctly on the table aliases. 

`DBT_INTERNAL_DEST` and `DBT_INTERNAL_SOURCE` are the standard aliases for the target table and temporary table, respectively, during the course of an incremental run using the merge strategy. 

I think a warning callout that since these predicates will be applied to the merge statement, you may need to use these aliases!